types/json.go

```diff
@@ -113,13 +113,19 @@ func (a JSONArray) String() string {
 	return o.String()
 }
 
+func (a *JSONArray) Value(i int) []byte {
```

Comment:
For better or worse, `AppendValues` takes `[]any`, so probably `Value` should return `any` (the unmarshalled bytes). Though I'm not sure `AppendValues` is doing the right thing here to be honest. What do you think?

Reply:
It also doesn't account for invalid values being appended...
I'll take a look

Reply:
But I think we can return `json.RawMessage` (~ `[]byte`) to be safe, as we can always say that unmarshaled is returned by `GetOneForMarshal`

Reply:
Moved ret type to `json.RawMessage` in 9a305de